docs/guides/admin/docs/configuration/security.jwt.md

```diff
@@ -1,18 +1,107 @@
+
+### Quick overview
+
+```json5
+{
+    // Standard claims
+    "exp": 1499863217, // required
+    "nbf": 1548632170, // optional
+
+    // User info
+    "username": "jose",
+    "name": "José Carreño Quiñones",
+    "email": "jose@example.com",
+
+    // Authorization, i.e. privileges the request/user should have
+    "roles": ["STUDIO", "API_EVENTS_VIEW"], // without `ROLE_` prefix
+    "oc": [
+        "read+customaction:e:d622b861-4264-4947-8db1-c754c5956433",
+        "write:s:4ed02421-144c-42a1-b98a-22e84f3ac691"
+    ]
+}
+```
+
+### Claims
+
+A claim being "required" means that the JWT provider (e.g. Tobira, LMS) has to include it. Opencast will reject all JWTs which lack required claims.
+
+- `exp` (expiration time): As [defined in the JWT RFC](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.4), a number timestamp in seconds since the unix epoch. JWTs with `exp < now()` are rejected by Opencast. This claim is *required*.
+
+- `nbf` (not before): As [defined in the JWT RFC](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.5), a number timestamp in seconds since the unix epoch. JWTs with `nbf > now()` are rejected by Opencast. This claim is optional.
+
+- *User Info*: information about the user that's possibly used by parts of Opencast. Some JWT use cases don't require user information at all (e.g. static file auth), while others do (e.g. Studio). Therefore, these claims are optional. (TODO: specify when this is stored as user reference)
+    - `username`: string. (TODO: explain how this is used to lookup existing users, if at all)
+    - `name`: display name, string.
+    - `email`: string.
+
+- *Authorization*: at least one of these claims has to be set, as otherwise no privileges are granted (which is the same as not sending a JWT at all).
+    - `roles`: array of strings, specifying roles to grant to the request/user directly. The `ROLE_` prefix must be omitted as Opencast adds it itself.
+    - `oc`: array of strings, grants access to single items in Opencast. The custom syntax is terse in order to keep JWTs as small as possible. Each string consists of three parts, which are separated by a colon (`:`):
+        - *Action(s)*: the first part specifies what actions are granted, corresponding to actions in the OC ACLs, e.g. `read` and `write`. Multiple actions can be specified by joining them with `+`. To keep the syntax unambigious, actions cannot contain `:` or `+` (ideally they are ASCII alphabetic only!). To further preserve space, some actions imply others:
+            - `write` implies `read` access
+            - `annotate` implies `read` access
+        - *Item Type*: The second part specifies what type of item this applies to:
+            - `e`: event
+            - `s`: series
+            - `p`: playlist
+        - *ID*: the final part specifies the ID of the item that is granted access to.
```

Comment:
Not sure, but IDs can probably also contain `:`. Shouldn't be a problem, if this string is parsed correctly.